src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -295,7 +296,8 @@ private AddEventsResponse parseAddEventsResponse(CloseableHttpResponse httpRespo
    * @param addEventsPayload  byte[] addEvents payload (post compression if compression is enabled).
    * @return Decompressed payload byte array.
    */
-  private byte[] getDecompressedPayload(byte[] addEventsPayload) {
+  @VisibleForTesting
+  protected byte[] getDecompressedPayload(byte[] addEventsPayload) {
```

Comment:
Remove `protected`.  Default visibility is enough.

Reply:
Removed.